update_exchange_rates.py

```diff
@@ -33,6 +33,10 @@ def main(config, push_to_server=False):
     usd_to_sek = c.get_rate('USD', 'SEK')
     eur_to_sek = c.get_rate('EUR', 'SEK')
 
+    # Inconsistent results for Euro after broken API was updated
```

Comment:
Very strange that occurs only for the eur to sek.

Reply:
I think the source API might be different.